src/net_processing.cpp

```diff
@@ -405,6 +405,10 @@ struct Peer {
     /** Protocol version indicated by the peer in its initial version message. */
     std::atomic<int> m_version{0};
 
+    /** Greates common protocol version that both we and the peer support. Set
```

Comment:
In 9873aa48a92ccf526762bde8d24911c3e81adb32: `Greates` and in the commit message `m_greates_common_version`